ckoga/SWAPI-Study-Sesh
# Fetch branch

### What's this PR do?
This PR merges a successful fetch call in componentsDidMount and a successful fetch call on button click to view characters.  
### Where should the reviewer start?
Take a look at App.js and look for the componentDidMount method here is our initial fetch to get all the Star Wars films we setState here which initiates the render.  7 movie cards should be rendered after this.
Look for the fetchChar method underneath componentDidMount and look over the fetch call made in this method.  This method is passed down to MovieCard.js as a prop to be triggered onClick.
Next go to MovieCard.js and check the onClick attribute that calls fetchChar.
### How should this be manually tested?
Pull down this branch run ```npm start``` once the page loads there should be 7 movie cards rendered to the page.  Each movie card has a ```View Characters``` button that will initiate a new fetch call to get that movie's characters.  Open up your dev tools and there should be console.logs showing the individual data points coming back.

### What are the relevant tickets?
issue #7 addresses this PR complete a good chunk of the functionality but need to finish getting all the data out of our fetch calls.
### Screenshots (if appropriate)

<img width="474" alt="Screen Shot 2019-12-05 at 3 27 48 PM" src="https://user-images.githubusercontent.com/45470456/70279521-e3dfe300-1773-11ea-8d45-4f634f08d16f.png">
